[IMP] inventory: rewrite lead times, add images

Felicious opened the pull request:
Rewrite [lead times](https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.html) doc for Odoo 14:
1. Update title, subheadings to sentence case
2. Rewrite to remove "you"s from the document
3. Rewrite navigational instructions for setting up lead times to reflect documentation guidelines
4. Wrap specific examples in admonition block
5. Create new images to supplement lead time scheduling
6. Add setup instructions to enable automatic scheduling of manufacturing orders

[Task number](https://www.odoo.com/web#id=3330609&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)
 
 Intended to be FW ported to all later versions. After this is published, I'll go and take newer screenshots for version 16.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/5095).

Felicious commented:
Hi @odoo/inventory-doc-review :wave: This lead times doc is ready for review! It's a complicated topic, so I made some graphics on my own and added them in afterwards. Because of this, there are things I'm not sure about and I'm looking for your input in particular on the lead times overview images on lines 146 and 226:
1. Any suggestions on the presentation of information and design suggestions? Does this graphic make sense, and how can I improve it, so it doesn't feel like a vague Figure from a textbook?
2. Should I adjust the examples in the following lead times section to reflect the overview image? (ie. Should the # of days in the vendor lead time in the image match the "Set up 10 days of vendor delivery lead time" in the example?)

And of course, very grateful for feedback on the rest of the doc! :smiley_cat:

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
 
-Supplier Lead Time
-------------------
+Simplify procurement by automatically determining dates to place orders from suppliers. Odoo
```
```suggestion
Automatically determining the dates on which to place orders from suppliers can help to simplify the procurement process. Odoo
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+calculates the supplier shipment *receipt date* and :abbr:`PO (Purchase Order)` deadline based on
+the required date the product is needed in the warehouse. By working backwards from the receipt
+date, vendor lead times and purchase security lead times are taken into account to determine the
+:abbr:`PO (Purchase Order)` deadline. This deadline  which is the date by which the order should be
+confirmed to ensure timely arrival by the expected receipt date.
```
```suggestion
:abbr:`PO (Purchase Order)` deadline. This deadline is the date by which the order should be
confirmed to ensure timely arrival by the expected receipt date.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+
+*Purchase security lead time* is set globally for the business in :menuselection:`Inventory app -->
+Configuration --> Settings`. On the configuration page, under the :guilabel:`Advanced Scheduling`
+heading, locate the box for :guilabel:`Security Lead Time for Purchase` to enable the feature by
+checking the box. Next, enter the desired number of calendar days. By configuring the security lead
```
```suggestion
heading, locate the box for :guilabel:`Security Lead Time for Purchase`, and click the checkbox to enable the feature. Next, enter the desired number of calendar days. By configuring the security lead
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+
+.. _inventory/management/planning/scheduled/manuf-LT:
+
+Manufacturing - lead times
```
```suggestion
Manufacturing lead times
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
 ==========================
 
-Manufacturing Lead Time
+Streamline the procurement of manufactured products by following Odoo's recommended manufacturing
+order (MO) start dates based on the required date the product is needed in the warehouse. Determine
```
```suggestion
Lead times can help simplify the procurement process for consumable materials and components used in manufactured products with bills of materials.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-Manufacturing Lead Time
+Streamline the procurement of manufactured products by following Odoo's recommended manufacturing
+order (MO) start dates based on the required date the product is needed in the warehouse. Determine
+the :abbr:`MO (Manufacturing Order)` deadline, which is the deadline to begin the manufacturing
+process to complete the product by the scheduled delivery date, by configuring manufacturing lead
+times and manufacturing security lead times.
```
```suggestion
The :abbr:`MO (Manufacturing Order)` deadline, which is the deadline to begin the manufacturing
process to complete the product by the scheduled delivery date, can be determined by configuring the manufacturing lead times and manufacturing security lead times.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-product. To specify it, open the *Inventory* tab of your product form
-and add the number of days the manufacturing takes.
+Configure the manufacturing lead time directly on the product form by navigating to
+:menuselection:`Manufacturing app --> Products --> Products` and select the desired product. In the
```
```suggestion
:menuselection:`Manufacturing app --> Products --> Products`, and select the desired product. In the
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+(Manufacturing Order)`**, is calculated as the *expected delivery date* - manufacturing lead time.
+This ensures that the manufacturing process begins on time to meet the delivery date. However it's
+important to note that lead times are based on calendar days. Lead times do not consider weekends,
+holidays, or work enter capacity.
```
```suggestion
holidays, or work center capacity.
```

brse-odoo replied:
A `dfn` tag could be fun here since people might be like "what is work center capacity?"! Example:

...holidays, or work center capacity. (:dfn:`the number of operations that can be performed at the work center simultaneously`)

Felicious replied:
This is a great idea! I just don't like how the definition tag renders on the website. (This is my very irrational hill I'll die on that I will not force on others LOL)

```suggestion
do not consider weekends, holidays, or :abbr:`work center capacity (the number of operations that
can be performed at the work center simultaneously)`.
```
Instead, I added (albeit incorrectly) an abbreviation tag because it looks more like what I think the definition tag should look while rendered on the webpage. I'll defer to @StraubCreative for the final decision.

StraubCreative replied:
hmm...think you'd want to use the `dfn` tag. Here's why:

if we use the `:abbr:` tag here subjectively, instead of the correct tag, it may lead to a bunch of whack-a-mole scenarios that will accumulate over time, which we'd have to fix later. It will also cause confusion among contributors who rely on the style guide and might invite unnecessary creativity around tag use.

@samueljlieber is looking into options around how this tag is rendered in the HTML, because yea, we do agree it could be improved.

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+
+*Manufacturing security lead time* is set globally for the business in :menuselection:`Manufacturing
+app --> Configuration --> Settings`. On the configuration page, under the :guilabel:`Planning`
+heading, locate the box for :guilabel:`Security Lead Time` to enable the feature by checking the
+box. Next, enter the desired number of calendar days. By configuring the security lead time, a
```
```suggestion
heading, locate the box for :guilabel:`Security Lead Time`, and click the checkbox to enable the feature. Next, enter the desired number of calendar days. By configuring the security lead time, a
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-.. image:: scheduled_dates/scheduled_dates_09.png
-    :align: center
-    :alt: View of the manufacturing lead time configuration from the product form
+Consider the following example to see how all the lead times work together to ensure timely order
```
```suggestion
See the following example to understand how all the lead times work together to ensure timely order
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-    :align: center
-    :alt: View of the manufacturing lead time configuration from the product form
+Consider the following example to see how all the lead times work together to ensure timely order
+fulfillment. Configured lead times:
```
```suggestion
fulfillment:
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-of the *MO* is **Commitment Date - Manufacturing Lead Time**. For
-example, the MO’s deadline start date for an order having a commitment
-date on the 10th of July is June 27th.
+-  Sales security lead time: 1 day
```
Do we wanna bold any of these bad boys in this list? So it would be like:

- **Manufacturing security lead time**: 2 days

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
 
-Global Example
-==============
+-  **September 14th**: Deadline to begin manufacturing. Calculated by subtracting  the manufacturing
```
```suggestion
-  **September 14th**: Deadline to begin manufacturing. Calculated by subtracting the manufacturing
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-==============
+-  **September 14th**: Deadline to begin manufacturing. Calculated by subtracting  the manufacturing
+   lead time of 3 days and the manufacturing security lead time of 2 days from the expected delivery
+   date of 9/19
```
```suggestion
   date of September 19th
```
Just to keep the date formatting consistent!

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
--  **September 19th**: the expected date on the delivery order form (1 day
-   of security lead time for sales)
+-  **September 19th**: :guilabel:`Scheduled Date` on the delivery order form indicates the updated
+   expected delivery date, which was originally set as 9/20. But the sales security lead time pushed
```
```suggestion
   expected delivery date, which was originally set as September 20th. But the sales security lead time pushed
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-   of security lead time for sales)
+-  **September 19th**: :guilabel:`Scheduled Date` on the delivery order form indicates the updated
+   expected delivery date, which was originally set as 9/20. But the sales security lead time pushed
+   the date forward by a day.
```
I think a nice closing sentence or paragraph after this would help to tie up the example or doc as a whole. What do you think?

Felicious replied:
Will think about what to write! Thanks for such a thorough review, Brandon (:

brse-odoo reviewed (commented):
Alright @Felicious, I've just finished reviewing this doc. Great job! Here are a few notes:
- Most of my suggestions were small grammar changes/fixes to clean up the language - as always, feel free to accept or ignore based on what you think
- I also left some "questions" in comment blocks for you to consider, just based on some things I think could help improve or clarify the doc
- I really like the graphics you made to help readers visualize all the moving parts in lead times. These are (IMO) a huge improvement upon the old graphics and I think you should definitely keep them

As always, if you have any questions about what I wrote, feel free to ping me. Thanks!

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-    :alt: View of the security lead time for sales configuration from the sales settings
+.. example::
+   Setting the :guilabel:`Security Lead Time for Sales` to `1.00` day pushes forward the
+   :guilabel:`Scheduled Date` of a :abbr:`DO (Delivery Order)` by a day. In that case, if a product
```
This was a great catch! Thanks for paying attention to the first instances of terms :smile_cat:

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
-of the *MO* is **Commitment Date - Manufacturing Lead Time**. For
-example, the MO’s deadline start date for an order having a commitment
-date on the 10th of July is June 27th.
+-  Sales security lead time: 1 day
+-  Manufacturing security lead time: 2 days
+-  Manufacturing lead time: 3 days
+-  Purchase security lead time: 1 day
+-  Vendor lead time: 4 days
```
```suggestion
-  **Sales security lead time**: 1 day
-  **Manufacturing security lead time**: 2 days
-  **Manufacturing lead time**: 3 days
-  **Purchase security lead time**: 1 day
-  **Vendor lead time**: 4 days
```

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+
+Establish a :abbr:`MO (Manufacturing Order)` deadline based on the *expected delivery date*,
+indicated in the :guilabel:`Scheduled Date` field of the :abbr:`DO (Delivery Order)`. The :abbr:`MO
+(Manufacturing Order)` deadline, which is the :guilabel:`Scheduled Date` field on the **:abbr:`MO
+(Manufacturing Order)`**, is calculated as the *expected delivery date* - manufacturing lead time.
```
I wanted to emphasize it, but it seems like the formatting isnt being applied. Will remove!

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+
+Establish a :abbr:`MO (Manufacturing Order)` deadline based on the *expected delivery date*,
+indicated in the :guilabel:`Scheduled Date` field of the :abbr:`DO (Delivery Order)`. The :abbr:`MO
+(Manufacturing Order)` deadline, which is the :guilabel:`Scheduled Date` field on the **:abbr:`MO
+(Manufacturing Order)`**, is calculated as the *expected delivery date* - manufacturing lead time.
```
minus sign :sweat_smile: 

I'll change it to a spelled out "subtracted by"

Felicious commented:
Hi @brse-odoo for such a thorough and careful review!!! 

I knew you were an amazing writer and editor but after you helped wrangle in the writing in this super long doc, I'm absolutely amazed by your work :blush: So grateful for the time and energy you spent helping me communicate lead times clearer (: I've addressed your suggestions and it's ready for another look :smile:

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,321 @@
+   - :ref:`Manufacturing planning <manufacturing/management/use_mps>`
+   - :ref:`Configure automatic MO scheduling with reordering rules
+     <inventory/management/reordering_rules>`
+   - call to action to enable weekends and holidays through a custom development? (is this salesy)
```
Asked someone from the sales team for a web page ! Waiting for the link to add here (:

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,330 @@
-can have different lead times but still need to be delivered, at once or
-separately. Fortunately, Odoo can help you handle these cases easily.
+For orders that include multiple products with different lead times, the lead times can be
+configured directly from the quotation itself. On a product form, click the :guilabel:`Other Info`
```
Are we on a product form here? Or a quotation?

brse-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,330 @@
+----------------
+
+To set a vendor lead time for orders arriving in the warehouse from a vendor location, begin by
+navigating to the product form through :menuselection:`Purchase app --> Products --> Products`.
```
```suggestion
navigating to a product form through :menuselection:`Purchase app --> Products --> Products`.
```
Since it can be any product

brse-odoo reviewed (commented):
@Felicious I've just finished my second review of this PR. Great job! Once again I don't have many comments at all, this doc is really tight now, too. Let me know if you have any other questions!

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,330 @@
+(Manufacturing Order)`, is calculated as the *expected delivery date* subtracted by the
+manufacturing lead time. This ensures that the manufacturing process begins on time to meet the
+delivery date. However it's important to note that lead times are based on calendar days. Lead times
+do not consider weekends, holidays, or :abbr:`work center capacity (the number of operations that
```
```suggestion
do not consider weekends, holidays, or work center capacity (:dfn:`the number of operations that
```

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,330 @@
+   - :ref:`Manufacturing planning <manufacturing/management/use_mps>`
+   - :ref:`Configure automatic MO scheduling with reordering rules
+     <inventory/management/reordering_rules>`
+   - call to action to enable weekends and holidays through a custom development? (is this salesy)
```
```suggestion
   
```

Felicious commented:
Hello @odoo/crm-doc-review ! This Inventory doc rewriting lead times is ready for your review! Excited to see how you're able to further improve the doc :blush: 

Thanks in advance!

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
-of the *MO* is **Commitment Date - Manufacturing Lead Time**. For
-example, the MO’s deadline start date for an order having a commitment
-date on the 10th of July is June 27th.
+*Purchase security lead time* is set globally for the business in :menuselection:`Inventory app -->
+Configuration --> Settings`. On the configuration page, under the :guilabel:`Advanced Scheduling`
+heading, locate the box for :guilabel:`Security Lead Time for Purchase`, and click the checkbox to
+enable the feature. Next, enter the desired number of calendar days. By configuring the security
+lead time, a buffer is set to account for potential delays in supplier deliveries.
```
```suggestion
*Purchase security lead time* is set globally for the business in :menuselection:`Inventory app -->
Configuration --> Settings`.

On the configuration page, under the :guilabel:`Advanced Scheduling` heading, locate the box for
:guilabel:`Security Lead Time for Purchase`, and click the checkbox to enable the feature.

Next, enter the desired number of calendar days. By configuring the security lead time, a buffer is
set to account for potential delays in supplier deliveries.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
-Security Lead Time
-------------------
+.. example::
+   Setting the :guilabel:`Security Lead Time for Purchase` to `2.00` days pushes back the
+   :guilabel:`Scheduled Date` of receipt by one day. In that case, if a product is initially
+   scheduled to arrive on April 6th, with a two-day security lead time, the new scheduled date for
+   the receipt would be April 8th.
```
```suggestion
   Setting the :guilabel:`Security Lead Time for Purchase` to `2.00` days, pushes the
   :guilabel:`Scheduled Date` of receipt back by one day. In that case, if a product is initially
   scheduled to arrive on April 6th, with a two-day security lead time, the new scheduled date for
   the receipt would be April 8th.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
-.. image:: scheduled_dates/scheduled_dates_10.png
-    :align: center
-    :alt: View of the security lead time for manufacturing from the manufacturing app settings
+Lead times can help simplify the procurement process for consumable materials and components used in
+manufactured products with bills of materials. The :abbr:`MO (Manufacturing Order)` deadline, which
+is the deadline to begin the manufacturing process to complete the product by the scheduled delivery
+date, can be determined by configuring the manufacturing lead times and manufacturing security lead
+times.
```
```suggestion
Lead times can help simplify the procurement process for consumable materials and components used in
manufactured products with bills of materials.

The :abbr:`MO (Manufacturing Order)` deadline, which is the deadline to begin the manufacturing
process to complete the product by the scheduled delivery date, can be determined by configuring the
manufacturing lead times and manufacturing security lead times.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+-----------------------
 
-Global Example
+Configure the manufacturing lead time directly on the product form by navigating to
+:menuselection:`Manufacturing app --> Products --> Products`, and select the desired product. In the
+:guilabel:`Inventory` tab of the product, specify the calendar days needed to the manufacture the
+product in the field, :guilabel:`Manufacturing Lead Time`.
```
```suggestion
Configure the manufacturing lead time directly on the product form, by navigating to
:menuselection:`Manufacturing app --> Products --> Products`, and select the desired product.

In the :guilabel:`Inventory` tab of the product, specify the calendar days needed to the manufacture
the product in the :guilabel:`Manufacturing Lead Time` field.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+      :align: center
+      :alt: View of the manufacturing lead time configuration from the product form.
+
+Establish a :abbr:`MO (Manufacturing Order)` deadline based on the *expected delivery date*,
+indicated in the :guilabel:`Scheduled Date` field of the :abbr:`DO (Delivery Order)`. The :abbr:`MO
+(Manufacturing Order)` deadline, which is the :guilabel:`Scheduled Date` field on the :abbr:`MO
+(Manufacturing Order)`, is calculated as the *expected delivery date* subtracted by the
+manufacturing lead time. This ensures that the manufacturing process begins on time to meet the
+delivery date.
```
```suggestion
Establish a :abbr:`MO (Manufacturing Order)` deadline, based on the *expected delivery date*,
indicated in the :guilabel:`Scheduled Date` field of the :abbr:`DO (Delivery Order)`.

The :abbr:`MO (Manufacturing Order)` deadline, which is the :guilabel:`Scheduled Date` field on the
:abbr:`MO (Manufacturing Order)`, is calculated as the *expected delivery date* subtracted by the
manufacturing lead time.

This ensures the manufacturing process begins on time, in order to meet the delivery date.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+manufacturing lead time. This ensures that the manufacturing process begins on time to meet the
+delivery date.
+
+However, it's important to note that lead times are based on calendar days. Lead times do not
+consider weekends, holidays, or work center capacity (:dfn:`the number of operations that can be
+performed at the work center simultaneously`).
```
```suggestion
However, it's important to note that lead times are based on calendar days. Lead times do **not**
consider weekends, holidays, or work center capacity (:dfn:`the number of operations that can be
performed at the work center simultaneously`).
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+
+.. example::
+   A product's scheduled shipment date on the :abbr:`DO (Delivery Order)` is August 15th. The
+   product requires 14 days to manufacture, so the latest date to start the :abbr:`MO (Manufacturing
+   Order)` to meet the commitment date is August 1st.
```
```suggestion
   product requires 14 days to manufacture. So, the latest date to start the :abbr:`MO
   (Manufacturing Order)` to meet the commitment date is August 1st.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+Manufacturing security lead time
+--------------------------------
+
+*Manufacturing security lead time* is set globally for the business in :menuselection:`Manufacturing
+app --> Configuration --> Settings`. Under the :guilabel:`Planning` heading, locate the box for
+:guilabel:`Security Lead Time`, and click the checkbox to enable the feature. Next, enter the
+desired number of calendar days. By configuring the security lead time, a buffer is set to account
+for potential delays in the manufacturing process.
```
```suggestion
*Manufacturing security lead time* is set globally for the business in :menuselection:`Manufacturing
app --> Configuration --> Settings`. Under the :guilabel:`Planning` heading, locate the box for
:guilabel:`Security Lead Time`, and click the checkbox to enable the feature.

Next, enter the desired number of calendar days. By configuring the security lead time, a buffer is
set to account for potential delays in the manufacturing process.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+   A product has a scheduled shipment date on the :abbr:`DO (Delivery Order)` set for August 15th.
+   The manufacturing lead time is 7 days, and manufacturing security lead time is 3 days. So, the
+   :guilabel:`Scheduled Date` on the :abbr:`MO (Manufacturing Order)` reflects the latest date to
+   begin the manufacturing order. In this example, the :abbr:`MO (Manufacturing Order)`'s planned
+   date is August 5th.
```
```suggestion
   begin the manufacturing order. In this example, the planned date on the :abbr:`MO (Manufacturing
   Order)` is August 5th.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+
+The customer places an order for a manufactured product on September 1st, and the scheduled delivery
+date from the warehouse is on September 20th. Odoo uses lead times and automated reordering rules to
+schedule the necessary operations based on the outgoing shipment delivery date, September 20th:
```
```suggestion
schedule the necessary operations, based on the outgoing shipment delivery date, September 20th:
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
+   :align: center
+   :alt: Show timeline of how lead times work together to schedule warehouse operations.
+
+-  **September 1st**: Sales order created, confirmed by salesperson
```
```suggestion
-  **September 1st**: Sales order created, confirmed by salesperson.
```

ksc-odoo reviewed (commented):
@Felicious just finished my peer review on this doc. great job! only a handful of minor adjustments to look at, but once those are taken care of, it should be ready to move on to the next stage. 👍

Felicious commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,335 @@
-
-Customer Lead Time
-------------------
+=========================
+Scheduling delivery dates
+=========================
```
Hmm, that's good to bring up. How about
```suggestion
=========================
Scheduled delivery dates
=========================
```

Felicious commented:
Hi @odoo/us-doc-review ! This passed 2 rounds of peer review and is ready for you! Thank you in advance (:

By the way, here's a [link to the Githib/runbot staging environment](https://runbot139.odoo.com/runbot/static/build/50128753-14-0/logs/build/html/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.html) for this doc for easy readability (:

samueljlieber commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,349 @@
+   :align: center
+   :alt: Show graphic of all lead times working together.
+
+-  **Customer lead time**: The default time frame for fulfilling customer orders. The customer lead
+   time is the number of days from the date the sales order (SO) is confirmed to the date the
+   products are shipped from the warehouse. This is also known as *delivery lead time*.
+   :ref:`[Skip to section] <inventory/management/customer-LT>`
+
+-  **Sales security lead time**: moves the *scheduled delivery date* forward by a specified number
+   of days. This serves as a buffer to allow the team ample time to prepare the outgoing shipment
+   earlier, considering the possibility of delays in the fulfillment process. :ref:`[Skip to
+   section] <inventory/management/sales-security-LT>`
+
+-  **Purchase lead time**: the number of days from the confirmation of a purchase order (PO) to the
+   receipt of products. It provides insight on the time it takes for products to arrive at the
+   warehouse, facilitating effective scheduling and planning of supplier deliveries :ref:`[Skip to
+   section] <inventory/management/purchase-LT>`
+
+-  **Purchase security lead time**: advances the order deadline on a :abbr:`PO (Purchase Order)` by
+   a specified number of days. This proactive approach of placing orders earlier mitigates the risk
+   of vendor or shipping delays. Thus, for products that are set to replenish to order, the need
+   appears on the *Replenishment report* earlier, according to the specified number of days.
+   :ref:`[Skip to section] <inventory/management/purchase-security-LT>`
+
+-  **Manufacturing lead time**: the number of days needed to complete a manufacturing order (MO)
+   from the date of confirmation. This lead time includes weekends (non-working hours in Odoo), and
+   is used to forecast an approximate production date for a finished good. :ref:`[Skip to section]
+   <inventory/management/manuf-LT>`
+
+-  **Manufacturing security lead time**: moves the scheduled date of the :abbr:`MO (Manufacturing
+   Order)` forward by a specified number of days. When used in conjunction with :ref:`replenish to
+   order <inventory/management/products/strategies>`, the security lead time makes the need appear
+   earlier on the replenishment report. :ref:`[Skip to section]
+   <inventory/management/manuf-security-LT>`
```
Lets add the internal references inline with the section titles to avoid ambiguous link names with extra formatting like "[Skip to Section]" 🙂

Also, these bullet points have an extra space between the point and the text.

Finally, I will suggest keeping the custom anchor text lowercase.
```suggestion
- :ref:`Customer lead time <inventory/management/customer-lt>`: The default time frame for
  fulfilling customer orders. The customer lead time is the number of days from the date the sales
  order (SO) is confirmed to the date the products are shipped from the warehouse. This is also
  known as *delivery lead time*.

- :ref:`Sales security lead time <inventory/management/sales-security-lt>`: moves the *scheduled
  delivery date* forward by a specified number of days. This serves as a buffer to allow the team
  ample time to prepare the outgoing shipment earlier, considering the possibility of delays in the
  fulfillment process.

- :ref:`Purchase lead time <inventory/management/purchase-lt>`: the number of days from the
  confirmation of a purchase order (PO) to the receipt of products. It provides insight on the time
  it takes for products to arrive at the warehouse, facilitating effective scheduling and planning
  of supplier deliveries

- :ref:`Purchase security lead time <inventory/management/purchase-security-lt>`: advances the order
  deadline on a :abbr:`PO (Purchase Order)` by a specified number of days. This proactive approach
  of placing orders earlier mitigates the risk of vendor or shipping delays. Thus, for products that
  are set to replenish to order, the need appears on the *Replenishment report* earlier, according
  to the specified number of days.

- :ref:`Manufacturing lead time <inventory/management/manuf-lt>`: the number of days needed to
  complete a manufacturing order (MO) from the date of confirmation. This lead time includes
  weekends (non-working hours in Odoo), and is used to forecast an approximate production date for a
  finished good.

- :ref:`Manufacturing security lead time <inventory/management/manuf-security-lt>`: moves the
  scheduled date of the :abbr:`MO (Manufacturing Order)` forward by a specified number of days. When
  used in conjunction with :ref:`replenish to order <inventory/management/products/strategies>`, the
  security lead time makes the need appear earlier on the replenishment report.
```

samueljlieber commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,349 @@
 
-To set this up, go to :menuselection:`Inventory --> Configuration -->
-Settings` and enable the feature *Security Lead Time for Sales*.
+   .. image:: scheduled_dates/customer-LT.png
```
Keep image filenames lowercase please :)
```suggestion
   .. image:: scheduled_dates/customer-lt.png
```

samueljlieber commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,349 @@
+See the following example to understand how all the lead times work together to ensure timely order
+fulfillment:
+
+-  **Sales security lead time**: 1 day
+-  **Manufacturing security lead time**: 2 days
+-  **Manufacturing lead time**: 3 days
+-  **Purchase security lead time**: 1 day
+-  **Vendor lead time**: 4 days
```
Remove extra spaces 
```suggestion
- **Sales security lead time**: 1 day
- **Manufacturing security lead time**: 2 days
- **Manufacturing lead time**: 3 days
- **Purchase security lead time**: 1 day
- **Vendor lead time**: 4 days
```

samueljlieber commented on content/applications/inventory_and_mrp/inventory/management/planning/scheduled_dates.rst:
```diff
@@ -1,262 +1,349 @@
+   :align: center
+   :alt: Show timeline of how lead times work together to schedule warehouse operations.
+
+-  **September 1st**: Sales order created, confirmed by salesperson.
+
+-  **September 9th**: Deadline to order components to ensure they arrive in time when manufacturing
+   begins (4-day supplier lead time).
+
+-  **September 13th**: Scheduled date of receipt for components. Initially, it was set to 9/14, but
+   the 1-day purchase security lead time pushed the date earlier by 1 day.
+
+-  **September 14th**: Deadline to begin manufacturing. Calculated by subtracting the manufacturing
+   lead time of 3 days, and the manufacturing security lead time of 2 days, from the expected
+   delivery date of September 19th.
+
+-  **September 19th**: :guilabel:`Scheduled Date` on the delivery order form indicates the updated
+   expected delivery date, which was originally set as September 20th. But the sales security lead
+   time pushed the date forward by a day.
```
Remove extra spaces
```suggestion
- **September 1st**: Sales order created, confirmed by salesperson.

- **September 9th**: Deadline to order components to ensure they arrive in time when manufacturing
  begins (4-day supplier lead time).

- **September 13th**: Scheduled date of receipt for components. Initially, it was set to 9/14, but
  the 1-day purchase security lead time pushed the date earlier by 1 day.

- **September 14th**: Deadline to begin manufacturing. Calculated by subtracting the manufacturing
  lead time of 3 days, and the manufacturing security lead time of 2 days, from the expected
  delivery date of September 19th.

- **September 19th**: :guilabel:`Scheduled Date` on the delivery order form indicates the updated
  expected delivery date, which was originally set as September 20th. But the sales security lead
  time pushed the date forward by a day.
```

samueljlieber reviewed (changes requested):
Hi @Felicious! This is a great PR, I only have a few technical corrections (see below). I will push up a commit with these corrections and approve. 🙂

samueljlieber commented:
Implemented my changes in https://github.com/odoo/documentation/commit/af833e9e1e04cb9671a110142970bc07c9a4834f however, the image filenames did not update, will make another commit to fix this

samueljlieber reviewed (approved):
Looks good to me @StraubCreative 👍

StraubCreative reviewed (approved):
@robodoo r+
